taskcluster/docker/base/Dockerfile

```diff
@@ -19,12 +19,9 @@ WORKDIR /builds/worker/
 #-- Configuration -----------------------------------------------------------------------------------------------------
 #----------------------------------------------------------------------------------------------------------------------
 
-ENV ANDROID_SDK_VERSION='3859397' \
-    ANDROID_SDK_ROOT='/builds/worker/android-sdk-linux' \
-    GRADLE_OPTS='-Xmx4096m -Dorg.gradle.daemon=false' \
+ENV CURL='curl --location --retry 5' \
```

Comment:
So far, we've been lucky `dl.google.com` was always available 🙂 Our script is now more resilient.